### PR TITLE
"Core Features" adaptation issue fixed

### PR DESCRIPTION
**Pull request by wolfboys**
navbar i18n support.

### CODE REVIEW

File: src/pages/team/languages.json [hunk 1]
```diff
@@ -183,7 +183,7 @@
         "email": "kirs@apache.org",
         "gitUrl": "https://github.com/CalvinKirs",
         "githubId": "Calvin Kirs",
-        "name": "Calvin Kirs"
+        "name": "CalvinKirs"
```
Comment (CalvinKirs) [inline src/pages/team/languages.json:186]: it should be the ID field does not allow spaces

Comment (wolfboys) [reply src/pages/team/languages.json]: ok, fixed.

**Comment by wolfboys** [comment]
@CalvinKirs PTAL again. thx.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**